src/media/css/header.styl

```diff
@@ -78,6 +78,12 @@ body {
         background-size: auto 37px;
         display: block;
         height: 47px;
+
+        @media $wider-desktop {
```

Comment:
Can we move this guy down to where the other `$wider-desktop` rules are? If there aren't any here feel free to create it and then just `.wordmark.desktop {...}`. Makes finding relevant overrides a little easier.